.eslintignore

```diff
@@ -0,0 +1,7 @@
+public/**
+raw/**
+**/dist/**
+**/node_modules/**
+static/**
```

Comment:
Is that OK to not add `'.cache/*',` in here?

Reply:
i added it. thank you